lib/commands/screenshots.js

```diff
@@ -108,4 +109,6 @@ commands.getElementScreenshot = async function (el) {
   return data;
 };
 
+commands.getViewportScreenshot = iosCommands.screenshot.getViewportScreenshot;
```

Comment:
you must use WebDriverAgent endpoints for getting the screenshot. The stuff from `appium-ios-driver` won't work here, since it's based on uiAutomation

Reply:
same as above, `getViewportScreenshot` in ios-driver only makes use of methods which we have overridden in xcuitest driver, so it will work. and i have proved it with a functional test!

Reply:
Please don't forget to test the implementation for landscape mode, since a lot of things are broken natively there.